target/linux/ipq806x/files/arch/arm/boot/dts/qcom-ipq8064-wg2600hp.dts

```diff
@@ -133,20 +133,71 @@
 	pinctrl-0 = <&mdio0_pins>;
 	pinctrl-names = "default";
 
-	ethernet-phy@0 {
-		reg = <0>;
-		qca,ar8327-initvals = <
-			0x00004 0x06000000  /* PAD0_MODE */
-			0x0000c 0x00080080  /* PAD6_MODE */
-			0x000e4 0x0006a545  /* MAC_POWER_SEL */
-			0x000e0 0xc74164de  /* SGMII_CTRL */
-			0x0007c 0x0000004e  /* PORT0_STATUS */
-			0x00094 0x0000004e  /* PORT6_STATUS */
-			>;
-	};
+	switch@10 {
+		compatible = "qca,qca8337";
+		#address-cells = <1>;
+		#size-cells = <0>;
+		reg = <0x10>;
+
+		qca8k,rgmii56_1_8v;
+
+		ports {
+			#address-cells = <1>;
+			#size-cells = <0>;
+
+			port@0 {
+				reg = <0>;
+				label = "cpu";
+				ethernet = <&gmac1>;
+				phy-mode = "rgmii-id";
+
+				fixed-link {
+					speed = <1000>;
+					full-duplex;
+				};
+			};
+
+			port@1 {
+				reg = <1>;
+				label = "lan1";
+			};
+
+			port@2 {
+				reg = <2>;
+				label = "lan2";
+			};
 
-	ethernet-phy@4 {
-		reg = <4>;
+			port@3 {
+				reg = <3>;
+				label = "lan3";
+			};
+
+			port@4 {
+				reg = <4>;
+				label = "lan4";
+			};
+
+			port@5 {
+				reg = <5>;
+				label = "wan";
+			};
```

Comment:
@Ansuel Hi, thank you for your great works!
In NEC Aterm WG2600HP, the adjustment of port mapping is needed like WG2600HP3.
(it also may also be needed for ASRock G10)

```
diff --git a/target/linux/ipq806x/files/arch/arm/boot/dts/qcom-ipq8064-wg2600hp.dts b/target/linux/ipq806x/files/arch/arm/boot/dts/qcom-ipq8064-wg2600hp.dts
index 04aace13a3..0babbc820d 100644
--- a/target/linux/ipq806x/files/arch/arm/boot/dts/qcom-ipq8064-wg2600hp.dts
+++ b/target/linux/ipq806x/files/arch/arm/boot/dts/qcom-ipq8064-wg2600hp.dts
@@ -159,27 +159,27 @@
 
 			port@1 {
 				reg = <1>;
-				label = "lan1";
+				label = "wan";
 			};
 
 			port@2 {
 				reg = <2>;
-				label = "lan2";
+				label = "lan1";
 			};
 
 			port@3 {
 				reg = <3>;
-				label = "lan3";
+				label = "lan2";
 			};
 
 			port@4 {
 				reg = <4>;
-				label = "lan4";
+				label = "lan3";
 			};
 
 			port@5 {
 				reg = <5>;
-				label = "wan";
+				label = "lan4";
 			};
 
 			/*
```

Reply:
I'm having some difficulty with checking the swconfig configuration in the 02_network file....
But I can't any reference about this. Can someone check with a wiki or some image the right port for every device?

Reply:
Someone can confirm about g10 ?

Reply:
cc: @CHKDSK88